app/services/browser_support.rb

```diff
@@ -0,0 +1,62 @@
+class BrowserSupport
+  @cache = LruRedux::Cache.new(1_000)
+
+  BROWSERSLIST_TO_BROWSER_MAP = {
+    and_chr: :chrome?,
```

Comment:
would we need to do like `browser.android? && browser.chrome?` instead of assuming all chrome = all android?

Reply:
Like if we supported proc (or took advantage of `Symbol#to_proc` for compatibility)

```suggestion
    and_chr: ->(browser) { browser.android? && browser.chrome? },
```

Reply:
> would we need to do like `browser.android? && browser.chrome?` instead of assuming all chrome = all android?

Added platform checking in 86c4df0c1.